apps/src/music/blockly/toolbox/toolboxBlocks.ts

```diff
@@ -11,8 +11,12 @@ import {
   TriggerStart,
 } from '../constants';
 
-const toolboxBlocks: {[blockType in BlockTypes | string]: BlockInfo} = {
+// Blockly's BlockInfo type with an additional 'text' field used for levelbuilder-facing English block text.
+type BlockInfoWithText = BlockInfo & {text: string};
+
+const toolboxBlocks: {[blockType in BlockTypes | string]: BlockInfoWithText} = {
   [BlockTypes.PLAY_SOUND]: {
+    text: 'Play Sound at Measure',
```

Comment:
Does Blockly ignore these values when interpreting this as a toolbox item?

Reply:
Hmm, good question. It's not part of the `BlockInfo` type so I'd assume, but probably worth using a more specific key to be safe.

Reply:
Updated to `levelbuilderText`